### PR TITLE
[Portal] Navigation breadcrumbs

### PR DESCRIPTION
**Pull request by Gustav-Eikaas**
# Description

Please include a summary of the change and which issue is fixed.
List any external dependencies that are required for this change.

Completes:
[AB#200562](https://dev.azure.com/Equinor/fa63ceea-8883-41e2-8823-a3b54e4be178/_workitems/edit/200562)

## Checklist:

-   [x] I have performed a self-review of my own code.
-   [x] I have linked my DevOps task using the AB# tag.
-   [x] My code is easy to read, and comments are added where needed.
-   [ ] My code is covered by tests.

### CODE REVIEW

File: src/components/TopBar/BreadCrumbs/Breadcrumbs.tsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import { Breadcrumbs } from '@equinor/eds-core-react';
+import { useLocation } from 'react-router';
+import styled from 'styled-components';
+
+import { readClientRegistry } from '../../../Core/Client/Functions';
+
+export const LocationBreadCrumbs = (): JSX.Element => {
+    const location = useLocation();
+
+    const createBreadCrumbs = () => {
```
Comment (Noggling) [inline src/components/TopBar/BreadCrumbs/Breadcrumbs.tsx:10]: useMemo eller useCallback

File: src/components/TopBar/BreadCrumbs/Breadcrumbs.tsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        if (tab) crumbs.push(tab[0].toUpperCase() + tab.substring(1));
+
+        return crumbs;
+    };
+
+    return (
+        <>
+            {createBreadCrumbs().map((s) => (
+                <>
+                    <BreadcrumbStyle>/</BreadcrumbStyle>
```
Comment (Noggling) [inline src/components/TopBar/BreadCrumbs/Breadcrumbs.tsx:33]: er det / eller > som skal brukes?

**Review by Noggling — APPROVED** [review]
Ser greit ut, men se på komentarer før du merger ;)

File: src/components/TopBar/BreadCrumbs/Breadcrumbs.tsx (hunk 2 above)
Comment (Gustav-Eikaas) [reply src/components/TopBar/BreadCrumbs/Breadcrumbs.tsx:33]: /

**Pull request merged**